src/max_flow/maxflow.cpp

```
@@ -40,17 +40,25 @@ PgrFlowGraph::PgrFlowGraph(
+        Which algorithm) {
     add_vertices(edges, source_vertices, sink_vertices);
 
     capacity = get(boost::edge_capacity, graph);
     rev = get(boost::edge_reverse, graph);
     residual_capacity = get(boost::edge_residual_capacity, graph);
 
-    if (algorithm == 1) {
-        insert_edges_push_relabel(edges);
-    } else {
-        insert_edges(edges);
+    switch (algorithm) {
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)